docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
+- New projects are created with a default priority level and are not active
+- Projects do not have a start or end date (start or end dates are specified on allocations, users and attributions)
```

Comment:
Allocations are not currently a concept in the policies or, thus, actual COGS system.  Certainly, Allocations might be desirable in the future but not there now.

Reply:
By "Allocation" I was referring to the "SystemAllocation" join table that you have in the current database schema, not to any consideration of policy. I'll change the wording.